x-pack/plugins/observability/public/pages/slos/hooks/use_url_search_state.ts

```diff
@@ -27,6 +28,7 @@ export interface SearchState {
 export const DEFAULT_STATE = {
   kqlQuery: '',
   page: 0,
+  perPage: 25,
```

Comment:
@shahzad31 Does it make sense to create a const for `25`, since you are using it in the `use_fetch_slo_list.ts` hook as well?